tests/run-regressionTest.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+set -e
+
+INPUT_DATA_PATH="$1"
+RESULT_PATH="$2"
+BINPATH="$3"
+FILENAME="$4"
+ABS_TOL="$5"
+REL_TOL="$6"
+SUMMARY_REGRESSION_TEST_COMMAND="$7"
+RESTART_REGRESSION_TEST_COMMAND="$8"
+INIT_REGRESSION_TEST_COMMAND="$9"
+EXE_NAME="${10}"
+TEST_ARGS="${@:{11}:100}"
```

Comment:
not this, was only the one above that needs parsing help (ie only when dereferencing the variable).